Sources/FINNFilterImplementation/FilterMarkets/FilterConfiguration.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol FilterConfiguration {
+    func handlesVerticalId(_ vertical: String) -> Bool
+    var preferenceFilterKeys: [FilterKey] { get }
+    var supportedFiltersKeys: [FilterKey] { get }
+    var mapFilterKey: FilterKey? { get }
+    func createFilterInfoFrom(filterData: FilterData) -> FilterInfoType?
```

Comment:
Maybe change it to something like `func createFilterInfoFrom(rangeFilterData: FilterData) -> FilterInfoType?` so it is clearer that it is only for filter data that has `isRange == true`

Reply:
Makes sense, I'll update it 👍